src/System.Private.ServiceModel/tools/scripts/WcfTestServerSetup.cmd

```diff
@@ -0,0 +1,191 @@
+@echo off
+setlocal
+
+set Id=%1
+set IdMaster=Master
+set WcfTestDir=c:\WCFTest
+set RepoHome=c:\git
+set CurrentRepo=%RepoHome%\wcf%Id%
```

Comment:
I'm a little surprised we get away with using % and not having EnableDelayedExpansion turned on.  I've often run into problems composing environment variables from other environment variables set in the same script.  Obviously it works for you -- I'm just observing I can't explain why.  Example, the other shared CoreFx scripts are moving to EnableDelayedExpansion and using ! instead of %.  Not blocking, just an observation.